Harivigneshkumar/Contact Form 1/index2.html

```diff
@@ -0,0 +1,197 @@
+    border-radius: 12px;
+    border-color: aliceblue;
+}
+.value:hover{
+    scale:131%;
+    background-color:#ffbdc7;
+}
+.query{
+    padding-right: 8px;
+    
+}
+    </style>
+</head>
+<body>
+    <div class="heading">
+        <h1>Contact Form (Editing Page)</h1>
+    </div><br>
+    <center>
+        <form action="index.html" method="get" target="_blank">
+    <div class="form1">
+        <label class="query">First Name :</label><input type="text" class="value" id="fname" placeholder="Enter your First Name"><br><br>
+        <label class="query">Last Name :</label><input type="text" class="value" id="lname" placeholder="Enter your Last Name"><br><br>
+        <label class="query">Mobile Number :</label><input type="text" class="value" id="mobilenumber" placeholder="Enter your Mobile Number"><br><br>
+        <label class="query">Date of Birth :</label><input type="date" class="value" id="dob" placeholder="Enter your date of birth"><br><br>
+        <label class="query">Gender :</label><select id="Gender" class="value">
+            <option value="male">male</option>
+            <option value="female">female </option></select><br><br>
+        <label class="query">Email :</label><input type="email" class="value" id="email" placeholder="Enter Your Email Id" required><br><br> 
+        <label class="query">Website :</label><input type="url" class="value" id="website" placeholder="Enter Your Website Name"><br><br>
+        <label class="query">Groups :</label><select id="Groups" class="value">
+            <option value="Important">Important</option>
+            <option value="Emergency">Emergency </option>
+            <option value="Not Assigned">Not Assigned </option></select><br><br>
+            <label class="query">Messager Accounts :</label>
+            <select id="messager" class="value">
+                <option value="Hangouts">Hangouts</option>
+                <option value="Skype">Skype </option>
+                <option value="Windows Live">Windows Live </option>
+                <option value="Yahoo">Yahoo </option>
+                <option value="QQ">QQ </option>
+            </select><br><br>
+        <label class="query">State :</label><select id="state" class="value">
+        <option value="TamilNadu">TamilNadu </option>
+        <option value="AndhraPradesh">AndhraPradesh </option>
+        <option value="Kerala">Kerala </option>
+        <option value="Karnataka">Karnataka </option>
+        </select><br></br>
+        <label class="query">Address :</label>
+        <input type="text" id="address" class="value" placeholder="Enter your Address"><br><br>
+        <label class="query">Work Info :</label>
+        <input type="text" class="value" id="work"placeholder="Enter your Job Title"><br><br>
+        <button type="submit" id="save1" onclick="savebutton()" class ="accept">Save</button>
+
+        <br><br><br></div></center></form>
+      <script>
+                     var obj5 = localStorage.getItem("testJSON");
```

Comment:
Move this Script code to separate JS file and provide link in the HTML page.
Indent the code properly in JS file.